assets/js/colorshare.js

```diff
@@ -0,0 +1,75 @@
+const span = document.getElementsByClassName("close")[0];
+const modal = document.getElementById("shareCodeModal");
+span.onclick = function () {
+    modal.style.display = "none";
+}
+window.onclick = function (event) {
+    if (event.target === modal) {
+        modal.style.display = "none";
+    }
+}
+
+document.getElementById("shareCodeSubmit").onclick = (
+    function () {
+        const url = 'api.php?code=' + document.getElementById("shareCode").value;
+        fetch(url)
+            .then(res => res.json())

```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ian76g/RailroadOnlineMapper/pullRequest?prid=8480274)
```suggestion
            .then((res) => res.json())
```